<launch>
	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find t_intern_assignment)/rviz_config/velodyne.rviz"/>

	<!-- main -->
	<node pkg="t_intern_assignment" name="image_to_pc" type="image_to_pc" output="screen">
		<param name="publish_rate" type="double" value="1"/>
		<param name="frame_id_name" type="string" value="/velodyne"/>
		<param name="file_path" type="string" value="/home/amsl/ozaki/job_hunting/toyota/assignment/datasets/bike_bicycle[5]/1"/>
	</node>
</launch>
